src/functions/asyncCredential/asyncCredentialHandler.ts

```diff
@@ -206,19 +207,70 @@ export async function lambdaHandler(
   const sessionServiceCreateSessionResult =
     await sessionService.createSession(sessionConfig);
 
+  const eventService = dependencies.eventService(config.SQS_QUEUE);
+
   if (sessionServiceCreateSessionResult.isError) {
+    const writeEventResult = await eventService.writeEvent({
+      eventName: "DCMAW_ASYNC_CRI_5XXERROR" as EventName,
+      sub,
+      sessionId,
+      ipAddress: "",
```

Comment:
Should this be removed if this is not an user interaction event or is there still value in having this field?

Reply:
Removed